lib/membrane_rtc_engine/tees/EngineTee.ex

```diff
@@ -0,0 +1,120 @@
+defmodule Membrane.RTC.Engine.EngineTee do
```

Comment:
I think that previous name was good. You have to change file name from `EngineTee.ex`  to `engine_tee.ex` or  just `tee.ex`. I think I would choose the second option and when refering to this tee I would  use `Engine.Tee`.